Code-the-Dream-School/dd-prac-team4-front
# PS-172 Add an error page and error boundary components

**Description:**

This pull request introduces a `PageNotFound` component for handling 404 errors in the React application. The component utilizes Material-UI for styling and theming, includes a responsive layout, and a back navigation button.

**Testing Steps:**

1. Type any non-existent path to render the `PageNotFound` component. For example: http://localhost:3000/homepage
2. Verify UI elements, theme consistency, and responsiveness.
3. Test the "Go to home page" button for navigation.
5. Ensure compatibility with various browsers.

<img width="708" alt="Screenshot 2023-09-08 at 10 43 37 PM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/2952900/fde29870-744f-494f-9649-37c906ee6ab1">
